hack/run-make-cmd.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
add some sanity checks to make it less fragile

```suggestion
set -o errexit
set -o nounset
set -o pipefail
set -o xtrace

REPO_ROOT=$(git rev-parse --show-toplevel)
cd ${REPO_ROOT}
```

Reply:
Thank you!